buildspecs/core.feature

```diff
@@ -110,8 +110,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="opt_inlineJsrs" value="true"/>
 		<flag id="opt_jitserver" value="false"/>
 		<flag id="opt_module" value="true"/>
+		<flag id="opt_methodHandle" value="true"/>
 		<flag id="opt_newObjectHash" value="true"/>
 		<flag id="opt_newRomClassBuilder" value="true"/>
+		<flag id="opt_openjdkMethodhandle" value="false"/>
```

Comment:
Perhaps this should be `opt_openjdkMethodHandle` (with a capital 'H') for consistency with `opt_methodHandle`?

Reply:
This syntax is already being used by 4+ developers across their branches. Worried that it may be a hassle for them to update it across all their changes. Also, the JPP tag is `OPENJDK_METHODHANDLES`, which has been merged. There is already inconsistency w.r.t. `OPENJDK_METHODHANDLES`. I would let the name stay as-is since we are prioritizing on functional completion.

Reply:
Fair enough.